connectomics/discfibers/ea_disctract.m

```diff
@@ -13,11 +13,18 @@
         shownegamount = [25 25] % two entries for right and left
         connthreshold = 20
         efieldthreshold = 200
-        statmetric = 1 % stats metric to use, 1 = ttest, 2 = correlations, 3 = OSS DBS pathway activations, 4 = Dice Coeff / VTAs for binary variables, 5 = reverse t-tests & e-fields for binary variables, 6 = show plain connections (no stats)
+        statmetric = 1 % stats metric to use, 1 = ttest, 2 = correlations, 3 = vacant, 4 = Dice Coeff / VTAs for binary variables, 5 = reverse t-tests & e-fields for binary variables, 6 = show plain connections (no stats)
```

Comment:
we do not need it anymore, because PAM (OSS-DBS) is not a separate test, but only a connectivity model. I hope we did remove all instances where 3 refers to OSS-DBS, but might make sense to keep for now for compatibility with old fiber filtering files